.github/workflows/agent.yml

```diff
@@ -218,3 +218,83 @@ jobs:
       - name: Push agent container orbcommunity
         if: github.event_name != 'pull_request'
         run: docker push -a orbcommunity/orb-agent-full
+
+  package-agent-with-otelcol-orb:
```

Comment:
Is it a temporary step until we verify otelcol-orb works as expected and then swap orb-agent-full instead?

Reply:
I would have both for now and then swap after testing.